packages/core/src/index.js

```diff
@@ -37,6 +39,27 @@ function createConfig (webpack, configSetters) {
 }
 
 /**
+ * Parses a semver string into an object
+ * @param {string} string
+ * @return {object}
+ */
+function parseVersionString (string) {
+  const semver = string.split('-')
+  const versions = semver[0]
+    .replace(/[^0-9.]/g, '')
```

Comment:
Why would you `.replace(/[^0-9.]/g, '')`? That means cutting off the major version, doesn't it? 🤔